docs/HowTo/Interact/APIs/Authentication.md

```diff
@@ -91,15 +89,65 @@ for HTTP and WS.
 
 Authentication tokens expire 5 minutes after being generated. It is necessary to generate a new authentication 
 token if access is required after token expiration.     
+
+## JWT Public Key Authentication
+
+Enable authentication from the command line and supply the public key of the external JWT token. The JWT token must use the `RS256` algorithm.
+
+### 1. Generate a Private and Public Key Pair
+
+The private and accompanying public key file must be in the `.pem` format.
```

Comment:
We need to mention that the user must use a RSA private key. And recommend that key be at least 2048 bits.

Reply:
Done

Reply:
Done